diff --git a/node_modules/@capacitor/android/capacitor/src/main/java/com/getcapacitor/CapacitorWebView.java b/node_modules/@capacitor/android/capacitor/src/main/java/com/getcapacitor/CapacitorWebView.java
index original..modified 100644
--- a/node_modules/@capacitor/android/capacitor/src/main/java/com/getcapacitor/CapacitorWebView.java
+++ b/node_modules/@capacitor/android/capacitor/src/main/java/com/getcapacitor/CapacitorWebView.java
@@ -63,12 +63,11 @@ public class CapacitorWebView extends WebView {
         boolean autoMargins = false;
         boolean forceMargins = configEdgeToEdge.equals("force");
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.VANILLA_ICE_CREAM && configEdgeToEdge.equals("auto")) {
+        if (configEdgeToEdge.equals("auto")) {
             TypedValue value = new TypedValue();
-            boolean foundOptOut = getContext().getTheme().resolveAttribute(android.R.attr.windowOptOutEdgeToEdgeEnforcement, value, true);
-            boolean optOutValue = value.data != 0; // value is set to -1 on true as of Android 15, so we have to do this.
-
-            autoMargins = !(foundOptOut && optOutValue);
+            boolean optOutValue = false;
+            
+            autoMargins = true;
         }
 
         if (forceMargins || autoMargins) {
